tools/run/solo5-run-virtio.sh

```diff
@@ -80,7 +80,13 @@ while true; do
         ;;
     -n)
         NETIF="$2"
-        ifconfig ${NETIF} >/dev/null || die "no such network interface: ${NETIF}"
+        # Check dependencies
+        type ip &>/dev/null ||
+            type ifconfig &>/dev/null ||
```

Comment:
thanks.  and yes, `type` seems to be POSIX (see http://pubs.opengroup.org/onlinepubs/9699919799/utilities/type.html).